code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -129,6 +129,12 @@
 	glass_desc = "The father of all refreshments."
 	shot_glass_icon_state = "shotglassclear"
 
+/datum/reagent/water/on_mob_life(mob/living/carbon/M)
+	current_cycle++
+	holder.remove_reagent(type, metabolization_rate / M.metabolism_efficiency)
+	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "hydration", /datum/mood_event/hydration, name)
+	M.AdjustUnconscious(-3, FALSE)
```

Comment:
you do know that this will only reduce unconsciousness times, not stun or sleeping times, yes?

Reply:
not sure why this was resolved but I think you're looking for `AdjustSleeping`

Reply:
Orange man says stun resist bad.

Reply:
what does this do then lol